### PR TITLE
enhance action throttle UI

### PR DESCRIPTION
**Pull request by ylwu-amzn**
1.Enhance action throttle UI
2.Change title to use sentence case

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: public/components/FormControls/FormikCheckbox/FormikCheckbox.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { render } from 'enzyme';
+import { Formik } from 'formik';
+
+import FormikCheckbox from './FormikCheckbox';
+
+describe.skip('FormikCheckbox', () => {
```
Comment (mihirsoni) [inline public/components/FormControls/FormikCheckbox/FormikCheckbox.test.js]: Why the test is being skipped ?

Comment (mihirsoni) [reply public/components/FormControls/FormikCheckbox/FormikCheckbox.test.js]: Also, could you add test cases to cover the branches which match snapshot.

Comment (ylwu-amzn) [reply public/components/FormControls/FormikCheckbox/FormikCheckbox.test.js]: Sorry, forgot to add it. Will change.

File: public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap [hunk 2]
```diff
@@ -20,7 +20,7 @@ exports[`WhereExpression renders 1`] = `
       <span
         class="euiExpression__value"
       >
-        all fields are included
```
Comment (dbbaughe) [inline public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap]: Seems to be undoing the change from Mihir's PR:
https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/pull/54/files

File: public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap [hunk 3]
```diff
@@ -20,7 +20,7 @@ exports[`WhereExpression renders 1`] = `
...
+        Select a field
```
Comment (dbbaughe) [inline public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap]: Seems to be undoing the change from Mihir's PR:
https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/pull/54/files

Comment (ylwu-amzn) [reply public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap]: Thanks, will merge it

Comment (mihirsoni) [reply public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap]: I think I replaced at wrong place. This should be fixed with new change.

Comment (ylwu-amzn) [reply public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap]: Published a new change, please help review

**Pull request merged**